packages/fieldset/src/LionFieldset.js

```diff
@@ -159,7 +161,15 @@ export class LionFieldset extends FormRegistrarMixin(
   }
 
   resetGroup() {
-    this.modelValue = this.resetModelValue;
+    this.formElementsArray.forEach(child => {
+      if (typeof child.resetGroup === 'function') {
+        child.resetGroup();
+      } else if (typeof child.reset === 'function') {
+        child.reset();
+      }
+    });
+
+    this.modelValue = this._initialModelValue;
```

Comment:
Since modelValue is a getter, this line is most likely not needed.
TODO: delete line and see if all functionality still works